<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>68</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Х</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>68</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Х</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>68</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>68</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPower">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>-5</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCoefficient">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>-500</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFreeMember">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>180</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>68</pointsize>
    </font>
   </property>
   <property name="text">
    <string>F(x) = </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>68</pointsize>
    </font>
   </property>
   <property name="text">
    <string>D(y) = [</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправить</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameConnected">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>721</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Статус подключения</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxLeftValue">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>-10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRightValue">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>-1000</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>290</y>
     <width>51</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>68</pointsize>
    </font>
   </property>
   <property name="text">
    <string>]</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
